.github/workflows/Pyodide.yml

```diff
@@ -44,8 +44,10 @@ jobs:
         version:
           - python: "3.10"
             pyodide-build: "0.22.1"
+            node: "16"
           - python: "3.11"
             pyodide-build: "0.25.1"
+            node: "18"
```

Comment:
Despite the fact that the emsdk version downloads and uses node 16, [Pyodide >= 0.25.0 requires node >=18](https://pyodide.org/en/stable/usage/index.html#node-js).

Without a separate installation of node, 0.25.1 the Python interpreter will fail to start for the 3.11 version here.

I added the node 16 install even though it isn't strictly necessary because it was less code than to create a condition on the job step.